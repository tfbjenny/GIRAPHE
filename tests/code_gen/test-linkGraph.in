node a = node("A");
node b = node("B");
node c = node("C");
node d = node("D");
node e = node("E");

print("Using + to link graph");

print("Link graphs: A -> 0$B ->3$E +  C -> 1$A + A -> 2$D");
print("Shared nodes: A");

graph g = a -> 0$b ->3$e +  c -> 1$a + a -> 2$d;

print(g);
