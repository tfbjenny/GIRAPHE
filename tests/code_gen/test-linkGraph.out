Using + to link graph
Link graphs: A -> 0$B ->3$E +  C -> 1$A + A -> 2$D
Shared nodes: A
--------------------------------------
Nodes:
node B
node E
node A
node C
node D

Edges:
edgeB ->E: 3
edgeA ->B: 0
edgeC ->A: 1
edgeA ->D: 2
--------------------------------------
